src/current/v24.3/set-up-logical-data-replication.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Set Up Logical Data Replication
+summary: Follow a tutorial to set up logical data replication between two clusters.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+In this tutorial, you will set up **logical data replication (LDR)** between two CockroachDB clusters. Both clusters are active and can serve traffic. You can apply the outlined steps to create a _unidirectional_ LDR stream from a source to a destination (cluster A to cluster B). 
```

Comment:
I think we should call out here somewhere that you will be replicating a table

Reply:
I have adjusted this whole first paragraph. PTAL